README.md

```diff
@@ -1 +1,27 @@
-# ShapesDemo-TypeSupport
\ No newline at end of file
+# ShapesDemo-TypeSupport
+
+Shapes Demo TypeSupport package to use with [eProsima Fast DDS Shapes Demo](https://www.github.com/eProsima/ShapesDemo) on ROS 2 installations.
+
+It can be built on a ROS 2 environment by running:
+
+    $ colcon build
+
+It can then be used to interact with topics created by Shapes Demo like so:
+
+    $ source install/setup.bash
+    $ ros2 topic echo /Square
```

Comment:
In general, I'd avoid the `$` and split the command and output into separate literal sections